source/includes/_endpoint-deck.md

```diff
@@ -0,0 +1,413 @@
+    "index": {
+        "/api/datasets/123/decks/123/slides/123/": {
+            "analysis_url": "/api/datasets/123/decks/123/slides/123/analyses/123/",
+            "subtitle": "z",
+            "display": {
+                "value": "table"
+            },
+            "title": "slide 1"
+        },
+        "/api/datasets/123/decks/123/slides/456/": {
+            "analysis_url": "/api/datasets/123/decks/123/slides/456/",
+            "subtitle": "",
+            "display": {
+                "value": "table"
+            },
+            "title": "slide 2"
+        }
+    },
+    "template": "{\"query\": {\"measures\": \"Object with keyed measures functions\", \"dimensions\": \"Array of variable references\", \"weight\": \"Optional weight variable URL\"}, \"display_settings\": {}, \"query_environment\": {\"filter\": \"Array of filter urls\"}}"
+}
+
+```
+
+Each tuple on the slides catalog contains the following keys:
+
+Name | Type | Description
+---- | ---- | -----------
+analysis_url | url | Points to the analysis contained on this slide
+title | string | Optional title for the slide
+subtitle | string | Optional subtitle for the slide
+display | object | Stores settings used to load the analysis
+
+#### POST
+
+To create a new slide an analysis has to be posted to the catalog. The payload, described in the [below](#analyses) section, should be
+wrapped as a shoji:entity.
+
+After successful creation, the server will return a 201 response with a Location header containing the URL of the newly created slide entity.
+
+Only a `query` field is required to create a new slide; other attributes are optional.
+
+
+Name | Type | Description
+---- | ---- | -----------
+query | object | Contains a valid analysis query, required
+title | string | Optional title for the slide
+subtitle | string | Optional subtitle for the slide
+display_settings | object | Contains a set of attribtues to be interpreted by the client to render and export the analysis
+query_environment | object | Contains the `weight` and `filter` applied during the analysis, they will be applied up on future evaluation/render/export
+
+### Entity
+
+`/datasets/223fd4/decks/slides/a126ce/`
+
+Each slide in the Slide Catalog contains reference to its analysis.
+
+#### GET
+
+```json
+{
+    "element": "shoji:entity",
+    "self": "/api/datasets/123/decks/123/slides/123/",
+    "catalogs": {
+        "analyses": "/api/datasets/123/decks/123/slides/123/analyses/"
+    },
+    "description": "Returns the detail information for a given slide",
+    "body": {
+        "deck_id": "123",
+        "subtitle": "z",
+        "title": "slide 1",
+        "analysis_url": "/api/datasets/123/decks/123/slides/123/analyses/123/",
+        "display": {
+            "value": "table"
+        },
+        "id": "123"
+    }
+}
+```
+
+#### DELETE
+
+Perform a DELETE request on the Slide entity resource to delete the slide and its analyses.
+
+### Order
+
+`/datasets/223fd4/decks/slides/flat/`
+
```

Comment:
Why does the URL for this say "flat" instead of just "order" like everywhere else?

In both the deck and slide order, the docs say that you shouldn't send nested groups, but is it forbidden?